app/javascript/mastodon/features/ui/components/columns_area.js

```diff
@@ -32,21 +32,29 @@ export default class ColumnsArea extends ImmutablePureComponent {
     children: PropTypes.node,
   };
 
-  handleRightSwipe = () => {
-    const previousLink = getPreviousLink(this.context.router.history.location.pathname);
-
-    if (previousLink) {
-      this.context.router.history.push(previousLink);
-    }
+  handleSwipe = (index) => {
+    window.requestAnimationFrame(() => {
+      window.requestAnimationFrame(() => {
```

Comment:
why nesting raF?

Reply:
Chrome and Safari have a bug where calling rAF once returns the current
frame instead of the next frame, so we need to call a double rAF here.
https://crbug.com/675795